test/jdk/java/security/CodeSource/Implies.java

```diff
@@ -51,14 +51,16 @@ public static void main(String[] args) throws Exception {
     }
 
     private static void testImplies(URL thisURL, URL thatURL, boolean result)
-        throws SecurityException
-    {
-        CodeSource thisCs =
-            new CodeSource(thisURL, (java.security.cert.Certificate[]) null);
-        CodeSource thatCs =
-            new CodeSource(thatURL, (java.security.cert.Certificate[]) null);
+            throws SecurityException {
+        CodeSource thisCs
+                = new CodeSource(thisURL, (java.security.cert.Certificate[]) null);
+        CodeSource thatCs
+                = new CodeSource(thatURL, (java.security.cert.Certificate[]) null);
         if (thisCs.implies(thatCs) != result) {
             throw new SecurityException("test failed");
```

Comment:
Can you add a more descriptive error message here, ex: `"CodeSource.implies() returned " + !result " instead of " + result`